components/jsonpath.rst

```diff
@@ -0,0 +1,94 @@
+The JsonPath Component
+==================
+
+    The JsonPath component queries JSON strings using JSONPath syntax.
+
+What is It?
+-----------
+
+The Symfony JsonPath component parses JSON strings to retrieve specifics values.
+
+`JSON`_, *JavaScript Object Notation*, is a lightweight data-interchange format. 
+It is easy for humans to read and write. JSON is a widely used format in REST API and data manipulation.
+
+.. tip::
+
+    Learn more about `JSONPath`_ specifications.
+
+Installation
+------------
+
+.. code-block:: terminal
+
+    $ composer require symfony/json-path
+
+.. include:: /components/require_autoload.rst.inc
+
+Why?
+----
+
+Fast
+~~~~
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
+Vestibulum id velit turpis. Sed vitae nisl placerat, rutrum ex nec, porta ante.
+Maecenas condimentum cursus risus at fringilla.
+
+Easy to read
+~~~~~~~~~~~
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
+Vestibulum id velit turpis. Sed vitae nisl placerat, rutrum ex nec, porta ante.
+Maecenas condimentum cursus risus at fringilla.
+
+Write powerful tests
+~~~~~~~~~~~~~~~~~~~~
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
+Vestibulum id velit turpis. Sed vitae nisl placerat, rutrum ex nec, porta ante.
+Maecenas condimentum cursus risus at fringilla.
+
+Ready-to-use functions
+~~~~~~~~~~~~~~~~~~~~
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
+Vestibulum id velit turpis. Sed vitae nisl placerat, rutrum ex nec, porta ante.
+Maecenas condimentum cursus risus at fringilla.
+
+.. _using-the-symfony-json-path-component:
+
+Using the Symfony JsonPath Component
+--------------------------------
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
+Vestibulum id velit turpis. Sed vitae nisl placerat, rutrum ex nec, porta ante.
+Maecenas condimentum cursus risus at fringilla.
+
+The JsonCrawler
+~~~~~~~~~~~~~~~~~~~~~
+
+(introduce the JsonCrawler and basic property access)
+
+Build JsonPath
+~~~~~~~~~~~~~~~~~~
+
+(introduce the JsonPath builder)
+-> list available method in the component (length, count, value, search, match, first, last)
+
+Advanced Usage
+---------------------
+
+Complex queries
+~~~~~~~~~~~~~~~~~~
+
+(show examples or more complex queries)
+
+Tests
+~~~~~~~~~~~~~~~~~~
+
+(show tests examples)
```

Comment:
Here you can talk about the newly merged PR: https://github.com/symfony/symfony-docs/issues/20977

Reply:
Awesome PR! Will be very useful, thanks